.gitattributes

```diff
@@ -0,0 +1,10 @@
+# Auto detect text files and perform LF normalization
+* text=auto
+
+# Explicitly declare text files
+*.py   text
+
+# Enforce platform-specific encodings
+*.bat  text eol=crlf
```

Comment:
Do you still plan to have a .bat file in-tree, or is more for personal use?

Reply:
This is mostly for futureproofing- I copied the gitattributes from the projectpiki pikmin2 decomp: [https://github.com/projectPiki/pikmin2/blob/main/.gitattributes](url)

Reply:
I don’t currently plan to have a bat file in the tree, but I was going to try making one for windows if the compare shell script failed to cooperate.